src/test/java/spark/staticfiles/StaticFilesTest.java

```diff
@@ -142,8 +143,13 @@ public void testDirectoryTraversalProtectionLocal() throws Exception {
         String path = "/" + URLEncoder.encode("..\\spark\\", "UTF-8") + "Spark.class";
         SparkTestUtil.UrlResponse response = doGet(path);
 
-        Assert.assertEquals(404, response.status);
-        Assert.assertEquals(NOT_FOUND_BRO, response.body);
+        // Attempt to access context above root is either a 400 or 404 depending on environment
```

Comment:
I fully agree. Could you create an issue from your comment so everyone can see it?